packages/sonarwhal/src/lib/config.ts

```diff
@@ -129,6 +142,14 @@ const updateConfigWithCommandLineValues = (config: UserConfig, actions: CLIOptio
         config.formatters = actions.formatters.split(',');
         debug(`Using formatters option provided from command line: ${actions.formatters}`);
     }
+
+    // If rules are provided, use them
+    if (actions && actions.rules) {
+        const rulesarray = actions.rules.split(',');
+
+        config.rules = buildRulesConfigFromRuleNames(rulesarray, 'warning');
```

Comment:
Shouldn't we use `error`?

Personally find useful to have the right status code, even for quick things.

Reply:
Most of the time, I use `warning` for rules,  as i did not want the program to exit. If you have a strong opinion it should be `error`, i will change it.

Reply:
Switched to `error`.